Signal/src/call/CallService.swift

```diff
@@ -449,9 +449,8 @@ private class SignalCallData: NSObject {
             self.handleFailedCall(failedCall: call, error: .obsoleteCall(description:"obsolete call in \(#function)"))
             return
         }
-
         guard callData.call == call else {
-            self.handleFailedCall(failedCall: call, error: .obsoleteCall(description:"obsolete call in \(#function)"))
+            Logger.warn("\(self.logTag) ignoring \(#function) for call other than current call")
```

Comment:
`handleFailedCall` kills the current call, if any, in an attempt to return to a known good state.  We shouldn't do this in this case which is out of the happy path but can be safely ignored.

Reply:
👍